src/pages/help-centre.js

```diff
@@ -1,13 +1,688 @@
+                                )}
+                            </ResultWrapper>
+                        </Container>
+                    </Backdrop>
+                </SearchSection>
+                <ArticleSection
+                    articles={articles}
+                    all_articles={all_articles}
+                    selected_article={selected_article}
+                    handleSelectArticle={this.handleSelectArticle}
+                    toggleSearch={this.toggleSearch}
+                />
+            </Layout>
+        )
+    }
+}
+
+// Eslint fails to recognize display names in React.memo
+// eslint-disable-next-line react/display-name
+const ArticleSection = React.memo(function ArticleSection({
+    articles,
+    all_articles,
+    handleSelectArticle,
+    selected_article,
+    toggleSearch,
+}) {
+    return (
+        <Container align="normal" direction="column">
+            {!selected_article && (
+                <LeftRightContainer padding="8rem 0">
+                    <ArticleList
+                        articles={articles}
+                        onClick={handleSelectArticle}
+                    />
+                </LeftRightContainer>
+            )}
+            {selected_article && (
+                <Article
+                    article={selected_article}
+                    all_articles={all_articles}
+                    onClick={handleSelectArticle}
+                    toggleSearch={toggleSearch}
+                />
+            )}
+        </Container>
+    )
+})
+ArticleSection.propTypes = {
+    all_articles: PropTypes.array,
+    article: PropTypes.object,
+    articles: PropTypes.array,
+    handleSelectArticle: PropTypes.func,
+    selected_article: PropTypes.object,
+    toggleSearch: PropTypes.func,
+}
+
+const ArticleContent = styled.div`
+    width: 60%;
+
+    > *:not(:last-child) {
+        margin-top: 0;
+        padding-bottom: 0.8rem;
+    }
+    ${Header} {
+        margin-bottom: 1.6rem;
+    }
+    ${Text} {
+        color: var(--color-black-3);
+        line-height: 1.5;
+    }
+`
+
+const StyledLink = styled(LocalizedLink)`
+    font-size: 2rem;
+    color: var(--color-red);
+
+    &::before {
+        content: '<';
+        display: inline-block;
+        margin-right: 4px;
+        font-weight: 300;
+        text-decoration: none;
+    }
+`
+
+const SmallSearchIcon = styled(SearchIcon)`
+    height: 27px;
+    width: 27px;
+
+    use {
+        fill: var(--color-green);
+    }
+    &:hover {
+        cursor: pointer;
+    }
+`
+
+const ListWrapper = styled.div`
+    ${Header} {
+        margin-bottom: 1.6rem;
+    }
+`
+
+const Article = ({ article, all_articles, onClick, toggleSearch }) => {
+    const related_articles = getRelatedArticles(all_articles, article)
+
+    return (
+        <>
+            <LeftRightContainer padding="8px 0 0 0">
+                <StyledLink to="/help-centre/">
+                    {localize('Back to Help topics')}
```

Comment:
should this one have underline text-decoration?

Reply:
Ah correct thanks it should have underline when it's hovered only